index.js

```diff
@@ -20,6 +20,7 @@ exports.command = require('./lib/command')
 exports.color = require('./lib/color')
 exports.debug = console.debug
 exports.mockConsole = console.mock
+exports.mockPrompt = prompt.mock
```

Comment:
I think you should just use the same mock we already have for everything else rather than having a mock just for the prompt